vulcan/_grade.py

```diff
@@ -80,7 +80,10 @@ def from_json(cls, j):
         id = j.get("Id")
         pracownik = Pracownik.from_json(j.get("Pracownik"))
         przedmiot = Przedmiot.from_json(j.get("Przedmiot"))
-        kategoria = KategoriaOceny.from_json(j.get("Kategoria"))
+        if (j.get("Kategoria")):
```

Comment:
In Python, you cannot use parentheses in `if` statements.
```suggestion
        if j.get("Kategoria"):
```

Reply:
I didn't actually knew that haha